manifests/apps/jupyter/deploy-python-model-quickstart.yaml

```diff
@@ -9,61 +9,61 @@ spec:
   appName: jupyterhub
   durationMinutes: 10
   icon: 'images/jupyterhub.svg'
-  description: How to deploy a Python model using Flask and OpenShift.
+  description: Deploy a Python model by using Flask and OpenShift
   prerequisites: [You completed the "Create a Jupyter notebook" quick start.]
   introduction: |-
     ### This quick start shows you how to deploy a Python model.
     Open Data Hub lets you run Jupyter notebooks in a scalable OpenShift hybrid cloud environment.
 
-    This quick start shows you how to take your model out of a Jupyter notebook and put it into a Flask application running on OpenShift Dedicated, for use as a development sandbox.
+    This quick start shows how to create a development sandbox by taking your model out of a Jupyter notebook and putting it into a Flask application that runs on OpenShift Dedicated. 
   tasks:
-    - title: Create a new GitHub repository using the s2i Flask template
+    - title: Create a new GitHub repository from the s2i Flask template
       description: |-
+       The s2i Flask template provides a simple s2i python project with notebooks included.
```

Comment:
And we are trying to migrate away from s2i, I think we might want to replace the whole tutorial